src/routes/DisciplineSelectionScreen.tsx

```diff
@@ -101,14 +101,16 @@ const DisciplineSelectionScreen = ({ route, navigation }: DisciplineSelectionScr
   return (
     <Root>
       <StatusBar backgroundColor='blue' barStyle='dark-content' />
-      <Title
-        title={discipline.title}
-        description={`${discipline.numberOfChildren} ${
-          discipline.numberOfChildren === 1 ? labels.home.unit : labels.home.units
-        }`}
-      />
       <ServerResponseHandler error={error} loading={loading} refresh={refresh}>
         <StyledList
+          ListHeaderComponent={
```

Comment:
Why this change?

Reply:
Otherwise the header would not scroll so you have a fixed header always visible, and the ScrollArea of the StyledList ist beneath.